mir_eval/io.py

```diff
@@ -463,6 +464,50 @@ def load_key(filename, delimiter=r'\s+'):
     return key_string
 
 
+def load_tempo(filename, delimiter=r'\s+'):
+    r"""Load tempo estimates from an annotation file in MIREX format.
+    The file should consist of three numeric columns: the first two
+    correspond to tempo estimates (in beats-per-minute), and the third
+    denotes the relative confidence of the first value compared to the
+    second (in the range [0, 1]). The file should contain only one row.
+
+    Parameters
+    ----------
+    filename : str
+        Path to the annotation file
+    delimiter : str
+        Separator regular expression.
+        By default, lines will be split by any amount of whitespace.
```

Comment:
Er, I stole that comment from the key loader... sounds like your fault :grin: 

But in general, it's the line(s) being split into columns.  I think the existing description is correct.